src/Test.hs

```diff
@@ -15,7 +15,16 @@ import           Diagrams.Prelude            hiding (interval, simulate, ui, (->
 
 
 background :: A.Animation Double I Rasterific V2 Double
-background = (ipure (rect 15 15 # fc white))
+background = (ipure (rect 55 35 # fc white))
+
+orbit :: (Num n, Ord n, Num a) => Double -> Double -> Double -> Active n I a -> Active n I a
+orbit xT yT (Active time f1) = 
```

Comment:
The type signature of `orbit` says it takes three doubles and an Active, but in the definition it only takes two doubles (`xT` and `yT`) and an Active.